keps/sig-storage/20200120-skip-permission-change.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Skip Volume Ownership Change
+authors:
+  - "@gnuified"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
+reviewers:
+  - "@msau42"
+  - "@liggit"
+  - "@tallclair"
+approvers:
+  - "@saad-ali"
+editor: TBD
+creation-date: 2020-01-20
+last-updated: 2020-01-20
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Skip Volume Ownership Change
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Graduation Criteria](#graduation-criteria)
+  - [Test Plan](#test-plan)
+  - [Monitoring](#monitoring)
+- [Implementation History](#implementation-history)
+- [Drawbacks [optional]](#drawbacks-optional)
+- [Alternatives [optional]](#alternatives-optional)
+- [Infrastructure Needed [optional]](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Summary
+
+Currently before a volume is bind-mounted inside a container the permissions on
+that volume are changed recursively to the provided fsGroup value.  This change
+in ownership can take an excessively long time to complete, especially for very
+large volumes (>=1TB) as well as a few other reasons detailed in [Motivation](#motivation).
```

Comment:
nit: Is the issue large volumes or the number of files/directories in the volume? Let's be very clear about that.

Reply:
I will fix this in a follow up